### ISSUE TITLE
Popup window with the message 'conf' when trying to leech/un-leech cards in the browser

### ISSUE THREAD

**Issue by iamjustkoi** [iamjustkoi/LeechToolkit#6]
**Bug/Issue Description**
When trying to Leech/Un-leech a card, a window pops up with the message 'conf' on it and the cards aren't updated (can't be leeched/un-leeched).

**To Reproduce**
Steps to reproduce:
1. Go to the Card/Note Browser window
2. Select Card(s)
3. Leech/Un-Leech

**Error Message**
```
File "concurrent.futures.thread", line 52, in run
File "aqt.operations", line 105, in wrapped_op
File "/Users/Winkel/Library/Application Support/Anki2/addons21/368380974/src/browser.py", line 200, in
start_collection_op(browser, lambda col: action_operation(col), tip_message, len(browser.selected_cards()))
File "/Users/Winkel/Library/Application Support/Anki2/addons21/368380974/src/browser.py", line 154, in action_operation
toolkit_configs = manager.get_all_configs()
File "/Users/Winkel/Library/Application Support/Anki2/addons21/368380974/src/config.py", line 48, in get_all_configs
config_id = self._mw.col.decks.get(deck_name_id.id)['conf']
KeyError: 'conf'
```

**Screenshots/Visuals**
![](https://user-images.githubusercontent.com/89978096/205300629-c2c9543e-f93e-4410-be88-85496667a6d7.jpg)

**System/Debug Info**
Debuginformationen:
Anki 2.1.54 (b6a7760c) Python 3.9.7 Qt 5.14.1 PyQt 5.14.1
Platform: Mac 12.6.1
Flags: frz=True ao=True sv=3
Add-ons, last update check: 2022-12-02 10:38:19
Möglicherweise beteiligte Erweiterungen: ⁨Leech Toolkit⁩

**Additional Context**
Originally posted in #4.

**Comment by iamjustkoi** [comment]
Created a potential fix for this in the recent hot-fix pre-release, don't know if it works on all systems, yet, though.
Will schedule it for the time being, either way.

**Comment by iamjustkoi** [comment]
Should be fixed in the most recent release! (v1.3.1)
